src/xcode/ENA/ENA/Source/Developer Menu/Features/DMCCLConfiguration/DMCCLConfigurationViewModel.swift

```diff
@@ -0,0 +1,102 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+#if !RELEASE
+
+import Foundation
+import UIKit
+import CertLogic
+
+final class DMCCLConfigurationViewModel {
+
+	// MARK: - Init
+
+	init(
+		restServiceProvider: RestServiceProviding
+	) {
+		self.restService = restServiceProvider
+	}
+
+	// MARK: - Internal
+
+	var viewController: UIViewController?
+	var refreshTableView: (IndexSet) -> Void = { _ in }
+
+	var numberOfSections: Int {
+		TableViewSections.allCases.count
+	}
+
+	func numberOfRows(in section: Int) -> Int {
+		guard TableViewSections.allCases.indices.contains(section) else {
+			return 0
+		}
+		// at the moment we assume one cell per section only
+		return 1
+	}
+	
+	func cellViewModel(by indexPath: IndexPath) -> Any {
+		guard let section = TableViewSections(rawValue: indexPath.section) else {
+			fatalError("Unknown cell requested - stop")
+		}
+
+		switch section {
+			
+		case .getCall:
+			return DMButtonCellViewModel(
+				text: "Call GET CLL Configuration",
+				textColor: .white,
+				backgroundColor: .enaColor(for: .buttonPrimary),
+				action: { [weak self] in
+					self?.restService.load(CCLConfigurationResource()) { result in
```

Comment:
Don't we need to do this through the CCLService so the update would also have an effect on the data in the app?

Reply:
if the CCLService holds the data, yeah, you are right. I will left this PR as draft so you can extend it as soon the CCLService is done.

Reply:
@nickguendling I think this topic can be closed, because setting the flag does not load any data. Instead the tester needs to restart the app.